CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,79 @@
+# jupyter-a11y-mgmt Repository Code of Conduct
+
+The `jupyter-a11y-mgmt` repository is a part of the Project Jupyter community. As such, it follows [Project Jupyter's Code of Conduct](https://jupyter.org/conduct/).
+
+Additionally, our team feels the need to specify our expectations around scope and unacceptable behavior as follows:
+
+## Scope and collaboration spaces
+
+This Code of Conduct applies to the following people in `jupyter-a11y-mgmt` repository:
+- admins of the online space
+- all community members
+- maintainers
+- reviewers
+- contributors
+
+While the `jupyter-a11y-mgmt` repository is our primary space, it may be necesary gather on other platforms to complete our work. The Code of Conduct applies to any online space where we announce we will be gathering to support work on the repository, whether meeting in real time or asynchorously.
+
+This includes:
+- Video and text chats
+- Official communications (like blog posts)
+- Interactions on GitHub whether through issues, pull requests, comments, review, or related
+
+## Inappropriate behavior
+
+Examples of unacceptable behavior include:
+
+- Harassment of any participants in any form
+- Deliberate intimidation, stalking, or following
+- Logging or taking screenshots of online activity for harassment purposes
+- Publishing others’ private information, such as a physical or electronic address, without explicit permission
```

Comment:
might want to add something like a.k.a doxing

Reply:
```suggestion
- Publishing others’ private information, such as a physical or electronic address, without explicit permission (a.k.a doxing)
```